### DISCUSSION TITLE
Custom error views not displayed after OrchardCore 1.8.3 update

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#16156] [Q&A] [answered]
Hi,
after updating OrchardCore to release 1.8.3 on my local environment, my application no longer displays my custom 404 error view like this:

![Screenshot 2024-05-26 at 08 45 53](https://github.com/OrchardCMS/OrchardCore/assets/61869142/0545e872-f607-4804-ad07-a8da50c8c92c)

and so it renders the default one:

![Screenshot 2024-05-26 at 08 46 21](https://github.com/OrchardCMS/OrchardCore/assets/61869142/275dfa33-9e96-477f-b7fc-5e127c1b8b6a)

My custom errror view is currently located on XXX.OrchardCore.Themes.Shared/Views/Shared/HttpError-Error.cshtml

The project containing this custom view is the following:

**XXX.OrchardCore.Themes.Shared.csproj**
```
<Project Sdk="Microsoft.NET.Sdk.Razor">
	<PropertyGroup>
		<TargetFramework>$(AspNetCoreTargetFramework)</TargetFramework>
		<AddRazorSupportForMvc>true</AddRazorSupportForMvc>
	</PropertyGroup>

	<ItemGroup>
		<FrameworkReference Include="Microsoft.AspNetCore.App" />
	</ItemGroup>

	<ItemGroup>
		<PackageReference Include="OrchardCore.Users.Abstractions" Version="$(OrchardCoreVersion)" />
		<PackageReference Include="OrchardCore.Users" Version="$(OrchardCoreVersion)" />
		<PackageReference Include="OrchardCore.Media" Version="$(OrchardCoreVersion)" />
		<PackageReference Include="OrchardCore.Module.Targets" Version="$(OrchardCoreVersion)" />
		<PackageReference Include="OrchardCore.ContentManagement" Version="$(OrchardCoreVersion)" />
		<PackageReference Include="OrchardCore.ResourceManagement" Version="$(OrchardCoreVersion)" />
		<PackageReference Include="OrchardCore.DisplayManagement" Version="$(OrchardCoreVersion)" />
		<PackageReference Include="OrchardCore.ReCaptcha.Core" Version="$(OrchardCoreVersion)" />
	</ItemGroup>

	<ItemGroup>
	  <ProjectReference Include="..\..\OrchardCore.Libs\XXX.OrchardCore.Manifest\XXX.OrchardCore.Manifest.csproj" />
	</ItemGroup>

</Project>
```

Both "OrchardCore.Diagnostics" and "OrchardCore.Templates" has been already enabled in my recipe app.

How could we solve this issue?

**Comment by Piedone** [comment, marked as answer]
The error pages are more flexible in 1.8.x, but this needs you to change your code a bit. See https://docs.orchardcore.net/en/latest/releases/1.8.0/#diagnostics-module

**Reply by DotCat1985** [reply]
Thank you for solving.

**Reply by Piedone** [reply]
Please mark it as an answer if it indeed solved it for you.

**Reply by DotCat1985** [reply]
> Please mark it as an answer if it indeed solved it for you.

Done.
